scripts/container_ops.sh

```diff
@@ -18,18 +19,24 @@ login() {
 # build "${dockerfile_path}"
 #
 build() {
-  DOCKER_FILE=${1:-infra/Dockerfile}
-  DOCKER_BUILDKIT=1 docker build --progress=plain \
+  local CONTAINER_IMG_TAG="$1"
+  local DOCKER_FILE="${2:-Dockerfile}"
+  DOCKER_BUILDKIT=0 docker build --progress=plain \
```

Comment:
DOCKER_BUILDKIT=0 ?
We intentionally enabled BUILDKIT to improve performance. We should have a parameter to enable/disable at least, but it should be enabled by default.

Reply:
It was turned off by mistake. I will enable this flag and later on will expand the script and add support for it.